packages/validator-js/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@sequelize/validator.js",
+  "version": "0.0.0-development",
+  "description": "Integration between validator.js and Sequelize",
+  "funding": [
+    {
+      "type": "opencollective",
+      "url": "https://opencollective.com/sequelize"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sequelize/sequelize.git"
+  },
+  "bugs": {
+    "url": "https://github.com/sequelize/sequelize/issues"
+  },
+  "main": "index.js",
+  "author": "",
```

Comment:
```suggestion
```

Reply:
There are some things we should add as well but I don't think they're for this PR.

Reply:
Yep, follow-up PR :)